lib/Optimizer/Lower.cpp

```diff
@@ -694,6 +694,141 @@ static void lowerTileNode(Function *F, const TileNode &TN) {
   TN.getResult().replaceAllUsesOfWith(IN);
 }
 
+std::pair<bool, std::vector<NodeInfo>>
+glow::getLoweredComponentInfo(Function *F, const NodeInfo &NI) {
```

Comment:
Add comments about what this does and to keep this in sync with actual lowering code.
Can we add tests to check that this is consistent with lowering code?

Reply:
There are some comments in `Optimizer.h` -- I added a little more to it.

It shouldn't be too bad to create a `LowerTest.cpp`, which creates nodes and then calls into both `lower()` and `getLoweredComponentInfo()` and makes sure they match.